website/docs/getting-started/setup/instance-management/supervisor.md

```diff
@@ -3,28 +3,45 @@ sidebar_position: 1
 ---
 # Supervisor
 
-The container runs multiple processes, including the Appsmith server, Nginx, MongoDB, etc., inside a single Docker container. These processes are started and managed by [Supervisor](http://supervisord.org).
+The container runs multiple processes, including the Appsmith server, Nginx, MongoDB, etc., inside a single Docker container. These processes are started, managed, and monitored by Supervisor.
 
-Supervisor comes with a web interface for managing the various processes, available at [http://localhost/supervisor](http://localhost/supervisor), as well as a command-line interface towards the same goal.
+[Supervisor](http://supervisord.org/) provides:
+* [Web Interface](#web-interface)
+* [Command line Interface](#command-line-interface) 
 
-:::info
-The credentials for supervisor access are defined by the environment variables `APPSMITH_SUPERVISOR_USER` and `APPSMITH_SUPERVISOR_PASSWORD` in the `stacks/configuration/docker.env` file.
-:::
+## Web interface
+The Supervisor web interface can be accessed at `http://localhost/supervisor`.
 
-Here's a screenshot of the web interface listing all the processes managed:
+<figure>
+  <img src="/img/appsmith_supervisord_ui.png" style= {{width:"700px", height:"auto"}} alt="The web interface listing of all the managed processes"/>
+  <figcaption align = "center"><i>The web interface listing of all the managed processes</i></figcaption>
+</figure>
 
-![](https://raw.githubusercontent.com/appsmithorg/appsmith/release/deploy/docker/images/appsmith\_supervisord\_ui.png)
+To set up the credentials for accessing the web interface, update the environment variables:
+1. Go to the `docker.env` file available in the `stacks/configuration` folder.
+2. Set the values for the below variables:
 
-The command-line interface can also be used to perform operations like restarting the Appsmith server, restarting Nginx, etc. For example, the following command (run in the installation folder) can be used to get the status of all running processes:
+```bash
+APPSMITH_SUPERVISOR_USER=<SUPERVISOR-USER>
+APPSMITH_SUPERVISOR_PASSWORD=<SUPERVISOR-PASSWORD>
+```
+
+3. Use the interface to manage the Appsmith server, Nginx, MongoDB, and more.
+
+## Command line interface
+The command line interface also helps in managing the Appsmith server, Nginx, MongoDB, and more. 
```

Comment:
Make sure we confirm that env variables need not be set up use the command line interface.

Reply:
The credentials through the env vars are not required for supervisorctl CLI. They are only required by browser